### ISSUE TITLE
Week 4 content

### ISSUE THREAD

**Issue by sprince0031** [sprince0031/ICT-Python-ML#4]
## Create content for week 4

- [ ] Data visualisation (Matplotlib, Seaborn)
    - [ ] Slide deck
    - [ ] Notebook - for live coding in video
    - [ ] Video(s)
- [ ] Scikit-learn, train/test split, simple classifier examples with sklearn
    - [ ] Slide deck
    - [ ] Notebook - for live coding in video
    - [ ] Video(s)
- [ ] Model evaluation - metrics like accuracy, precision, recall and f1 score
    - [ ] Slide deck
    - [ ] Notebook - for live coding in video
    - [ ] Video(s)

> [!NOTE]
> [Module outline document](https://github.com/sprince0031/ICT-Python-ML/blob/main/ICT%20Python%20and%20ML%20Foundations%20-%20Revised%20Module%20Outline.pdf)
